spring-integration-core/src/main/java/org/springframework/integration/history/MessageHistoryConfigurer.java

```diff
@@ -45,7 +45,7 @@
  *
  * @since 2.0
  */
-@ManagedResource
+@IntegrationManagedResource
```

Comment:
And why have you removed `@ManagedResource` here ?
I may not be interested in handlers/channels metrics, but want to manage this component from JMX too